ios/RNZoomUs.m

```diff
@@ -20,9 +24,13 @@ @implementation RNZoomUs
 - (instancetype)init {
   if (self = [super init]) {
     isInitialized = NO;
+    shouldAutoConnectAudio = NO;
+    enableCustomMeeting = NO;
+    showVideoPreviewWhenJoinMeetingDisabled = NO;
```

Comment:
Should be named `disableShowVideoPreviewWhenJoinMeetingDisabled` and should be `YES` on default I guess?